src/components/PrimaryNav/_primary-nav.scss

```diff
@@ -0,0 +1,57 @@
+@import "vanilla-framework/scss/settings_colors";
+
+.p-primary-nav {
+  a,
+  a:hover {
+    color: $color-light;
+  }
+
+  &__header {
+    display: grid;
+    grid-template-columns: 1fr 40px;
+    margin-bottom: -1rem;
```

Comment:
Why not set the height with `vh`?

Reply:
Viewport height cannot be relied upon here to give a consistent height for the navigation header.

Reply:
I may not have the background to understand why, but if it's not clear maybe a comment should be added here for future dev's who think to make the same change.